anisoap/representations/radial_basis.py

```diff
@@ -155,15 +190,168 @@ def __init__(
             else:
                 raise ValueError("`max_radial` must be None, int, or list of int")
 
+    # Get number of radial functions
+    def get_num_radial_functions(self):
+        return self.num_radial_functions
+
+    def plot_basis(self, n_r=100):
```

Comment:
@arthur-lin1027 docstring :)

Reply:
done for plot_basis and get_num_radial_functions